test/integration/scheduler/priorities_test.go

```diff
@@ -172,3 +173,65 @@ func TestPodAffinity(t *testing.T) {
 	}
 	t.Errorf("Pod %v got scheduled on an unexpected node: %v.", podName, pod.Spec.NodeName)
 }
+
+// TestImageLocality verifies that the scheduler's image locality priority function
+// works correctly, i.e., the pod gets scheduled to the node where its container images are ready.
+func TestImageLocality(t *testing.T) {
+	context := initTest(t, "image-locality")
+	defer cleanupTest(t, context)
+
+	// Add a few nodes.
+	_, err := createNodes(context.clientSet, "testnode", nil, 10)
+	if err != nil {
+		t.Fatalf("cannot create nodes: %v", err)
+	}
+
+	// We use a fake large image as the test image used by the pod, which has relatively large image size.
+	// TODO: replace this image with a LargeImage dedicated to testing image locality once the LargeImage is added
```

Comment:
Are your suggesting using a large e2e test image here? I think in integration test, it seems we will always have to use fake image like this.

Reply:
I agree. The intention was that if there were a large-image in the e2e images (imageutils) then we would be able to simply use it - but this does not justify a TODO.

It will be removed. Thanks!